packages/react/telemetry.yml

```diff
@@ -0,0 +1,815 @@
+# yaml-language-server: $schema=https://unpkg.com/@ibm/telemetry-config-schema@v0/dist/config.schema.json
```

Comment:
This config was generated with a script by @francinelucca to get all prop and prop values that we want to collect. We're still finishing the script to use when instrumenting other packages. Once that script is done, _maybe_ we could add that to a CI/CD script to automate this config's upkeep as components, component props, and supported component prop values change.